### PR TITLE
Update README to use Fishjam (rebranding)

### PR DESCRIPTION
**Pull request by karkakol**
Update README file to use new name - Fishjam

### CODE REVIEW

**Review by kamil-stasiak — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,17 +33,17 @@ Add JellyfishClientSDK dependency to your project.
...
+| Services    | [Videoroom](https://github.com/jellyfish-dev/jellyfish_videoroom) - an example videoconferencing app written in elixir <br/> [Dashboard](https://github.com/jellyfish-dev/jellyfish-dashboard) - an internal tool used to showcase Fishjam's capabilities    |
+| Resources   | [Fishjam Book](https://jellyfish-dev.github.io/book/) - theory of the framework, [Docs](https://jellyfish-dev.github.io/jellyfish-docs/), [Tutorials](https://github.com/jellyfish-dev/jellyfish-clients-tutorials)                                          |
+| Membrane    | Fishjam is based on [Membrane](https://membrane.stream/), [Discord](https://discord.gg/nwnfVSY)                                                                                                                                                              |
+| Compositor  | [Compositor](https://github.com/membraneframework/membrane_video_compositor_plugin) - Fishjam plugin to transform video                                                                                                                                      |
```
Comment (mironiasty) [inline README.md:45]: Are you sure that this should be changed? Compositor is part of Memebrane

**Review by mironiasty — COMMENTED** [review]
As organisation name is updated, you can use new links to repo now too

**Review by kamil-stasiak — CHANGES_REQUESTED** [review]
Organization and all repositories names changed so we're ready to replace every link

File: README.md (hunk 1 above)
Comment (karkakol) [reply README.md:45]: yep, it should go back

**Review by kamil-stasiak — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+[Fishjam](https://github.com/fishjam-dev/fishjam) Client library for iOS apps written in Swift.
 
 ## Components
 
 The repository consists of 2 separapable components:
 
-- `JellyfishClientSdk` - Jellyfish client fully compatible with `Jellyfish`, responsible for exchaning media events and receiving media streams which then are presented to the user
-- `JellyfishCLientDemo` - Demo application utilizing `Jellyfish` client
+- `FishjamClientSdk` - Fishjam client fully compatible with `Fishjam`, responsible for exchaning media events and receiving media streams which then are presented to the user
+- `FishjamCLientDemo` - Demo application utilizing `Fishjam` client
```
Comment (kamil-stasiak) [inline README.md:10]: Right now we don't have `FishjamClientSdk` and `FishjamCLientDemo`. I'm not sure whether we should change it now

Comment (karkakol) [reply README.md]: U right.

Comment (karkakol) [reply README.md]: Gone back. It will be changed later.

**Review by balins — APPROVED** [review]

**Pull request merged**